### DISCUSSION TITLE
是否有计划增加生成PDF的功能？

### DISCUSSION THREAD

**Discussion by alucadoli** [xisuo67/XHS-Spider#107] [Ideas]
下载完成后，自动生成PDF文件

**Comment by xisuo67** [comment]
不太理解这个需求



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年9月14日(周四) 上午9:21
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [xisuo67/XHS-Spider] 是否有计划增加生成PDF的功能？ (Discussion #107)




 
下载完成后，自动生成PDF文件

**Comment by alucadoli** [comment]
比如说，我下载了一篇小红书的文章，可能会有几个图片文件，能否按照小红书的图片显示顺序，把这些图片文件生成一个PDF文件，方便阅读？
之前我用Python写过类似的功能。

**Comment by xisuo67** [comment]
是个不错的想法，如果用户需求高的话，可以考虑加进去



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年9月14日(周四) 中午1:01
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [xisuo67/XHS-Spider] 是否有计划增加生成PDF的功能？ (Discussion #107)




 
比如说，我下载了一篇小红书的文章，可能会有几个图片文件，能否按照小红书的图片显示顺序，把这些图片文件生成一个PDF文件，方便阅读？
 之前我用Python写过类似的功能。
